crates/engine/codegen/domain/schema.graphql

```diff
@@ -267,6 +269,11 @@ type VirtualSubgraph @meta(module: "subgraph/virtual_") @indexed(id_size: "u16")
   schema_directives: [TypeSystemDirective!]!
 }
 
+enum SubscriptionsProtocol @meta(module: "subgraph/subscriptions_protocol") {
```

Comment:
You didn't need to create an enum here, you could have created the enum outside and used a scalar here, but why not^^

Reply:
Aah, I see what you mean, should have thought of that. I'll revert that part then, better not add unnecessary code.

Reply:
It was still a nice refresher on the codegen :)

Reply:
basically a scalar is used whenever it's something external. You can add the `@copy` also which will avoid the generation of a method